src/views/Correction/FmCorrection.vue

```diff
@@ -97,6 +103,21 @@ export default class FmCorrection extends Mixins(CommonMixin, FeeMixin, FilingTe
       // pre-select No Fee option
       this.setStaffPayment({ option: StaffPaymentOptions.NO_FEE })
 
+      const mailingAddress = this.getOriginalOfficeAddresses.businessOffice?.mailingAddress
+
+      this.setCompletingParty({
+        firstName: this.getUserFirstName,
+        lastName: this.getUserLastName,
+        mailingAddress: {
+          addressCity: mailingAddress?.addressCity,
+          addressCountry: mailingAddress?.addressCountry,
+          addressRegion: mailingAddress?.addressRegion,
+          postalCode: mailingAddress?.postalCode,
+          streetAddress: mailingAddress?.streetAddress,
+          streetAddressAdditional: mailingAddress?.streetAddressAdditional
+        }
+      } as CompletingPartyIF)
+
```

Comment:
First, I think it's a risky assumption that the completing party's address is the same as the business office.

Second, the other filings that manage completing party data don't do this.

Why are you doing this?

Reply:
This is part of the ticket's requirement: "For corrections triggered by clients, need to display the Completing Party, name, email and address, editable, pre-populated with the logged in user values.". Maybe I am missing something here.

Reply:
How do the other filings in this UI handle this requirement?

Reply:
Will this not prefill for both client triggered corrections and staff triggered corrections?

Reply:
The other filings in this UI doesn't render pre-populated completing party with the logged in user values.

Reply:
Why doesn't this code prefill things? Maybe it needs to be fixed.

https://github.com/bcgov/business-edit-ui/blob/4b3892439710e413f6e0cb98d29b27d8bd9b8cdb/src/App.vue#L534

Reply:
Yeah, this code doesn't load completing party.

Reply:
Does it work in other filings and is only broken for correction filings? Or only when staff?

Reply:
This doesn't work with other filings either. This code doesn't load anything into `setCompletingParty`, and I think we can ask about it during tomorrow's standup.

Reply:
Does this code work if you're logged in as not staff?

Reply:
Let me try. Hold on.

Reply:
I don't know where we'd get the name to pre-populate if we are logged in as staff -- a good question to ask tomorrow.

Reply:
I logged in as a client user and got it tested on a change filing. It actually loads the pre-populated values for the completing party component, which is not editable for client users.

Reply:
OK, so we need to figure out what we want when logged in as staff (and whether we can do it).

Reply:
I think I didn't read this line `if (!this.isRoleStaff)` in the code below:
https://github.com/bcgov/business-edit-ui/blob/4b3892439710e413f6e0cb98d29b27d8bd9b8cdb/src/App.vue#L665-L669

You're right. I'll mention it during the standup.

Reply:
> Will this not prefill for both client triggered corrections and staff triggered corrections?

The code is now restored/removed, but that code will prefill for either client or staff triggered corrections. We'll see whether we should prefill correction filing for users logged in as staff.

Reply:
Wait - "client or staff triggered corrections" can only be done by staff. (I'm not sure if "SBC Staff" will ever do corrections, but if they do then they have non-idir login so this code may inadvertently use their info. This is another question to ask.)

Does this mean we cannot prefill any of this data for a correction?

Reply:
We currently cannot prefill any of this for
1. a correction filing
2. any other type of filings, i.e. change filing, if logged in as a staff user.

Reply:
It was confirmed from Sarah that from the business point of view, completing party in any type of filing should be left empty if the logged in user is not the owner/account user. So in our case, completing party is not pre-filled for staff users, and the current/original implementation is correct.